frontend/src/components/Profile/MyProfile.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <v-card class="pa-2 ma-5">
+    <v-card
+      class="ma-4"
+      height="250"
+      width="250"
+      :img="this.$store.getters.avatar"
```

Comment:
Avatar is blank for me when user does not have an avatar, instead of the default

(Works correctly if an avatar is set)

Reply:
Could you check that `git-lfs` is installed and that the git hooks are running? Maybe check if you have set their permissions so that they are executable by your current user.

Reply:
I will double check. I take that to mean this works correctly for you?

I'm still getting a handle on git-lfs, but does it make sense that the `king-pong` image would appear correctly in the ProfileCard in the AppBar, but not on the ProfileView itself?

Reply:
You were right, it was not a `git-lfs` error. I pushed a correction on this branch.